### ISSUE TITLE
Unable to solve my SHFB project build failing

### ISSUE THREAD

**Issue by RobThree** [EWSoftware/SHFB#323]
![image](https://cloud.githubusercontent.com/assets/607888/17402469/b42bebda-5a53-11e6-9aba-a4a39305175a.png)

I checked [the documentation](https://ewsoftware.github.io/SHFB/html/50c2326c-5f1d-4615-9ee6-ad1d4952eded.htm) and looked at the logs:

These are the errors just prior:

```
BuildAssembler : warning : ShowMissingComponent: [G:] Missing <summary> documentation [D:\git\Repos\NUlid\NUlid.Documentation\Help\Working\BuildTopics.proj]
BuildAssembler : warning : SharedContentComponent: [G:] Missing shared content item. Tag: 'include'; Id:'topicTitle_'. [D:\git\Repos\NUlid\NUlid.Documentation\Help\Working\BuildTopics.proj]
BuildAssembler : warning : SharedContentComponent: [G:] Missing shared content item. Tag: 'include'; Id:'topicTitle_'. [D:\git\Repos\NUlid\NUlid.Documentation\Help\Working\BuildTopics.proj]
BuildAssembler : error :  [D:\git\Repos\NUlid\NUlid.Documentation\Help\Working\BuildTopics.proj]
BuildAssembler : error : SharedContentComponent: [G:] Error replacing item.  Root document element encountered. [D:\git\Repos\NUlid\NUlid.Documentation\Help\Working\BuildTopics.proj]
```

I don't understand what's going on. It was building fine until at some point this just started happening and I can't figure out where the problem is. Attached is the [:paperclip: LastBuild.log](https://github.com/EWSoftware/SHFB/files/401670/LastBuild.zip);

You can reproduce this by checking out [this specific commit](https://github.com/RobThree/NUlid/tree/53999980b243d8b37959d7a81a1658068016f506) of my project and building the solution in `Release` mode.

SHFB is, to me, somewhat notorious in 'unclear' or 'confusing' errors but usually I can get them solved. This time I'm baffled...

Oh, also relevant: I'm using VS2015, SP3 and all updates installed + [SHFB latest (v2016.5.28.0)](https://github.com/EWSoftware/SHFB/releases/tag/v2016.5.28.0).

Any help would be greatly appreciated! :kissing:

**Comment by RobThree** [comment]
Crap :hankey: :grin: 

Turns out someone experienced the same problem as me (why my initial Google and GitHub queries didn't turn up #323 is beyond me). But it turns out that:

![image](https://cloud.githubusercontent.com/assets/607888/17402897/b0b2ff0a-5a55-11e6-8bab-6d18d12432ef.png)

**Either** _both_ checkboxes need to be checkd **or** _both_ checkboxes need to be unchecked.

**Comment by EWSoftware** [comment]
It's a problem with the presentation styles and is fixed for the next release (related #316).
